hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RBFConfigKeys.java

```diff
@@ -96,6 +96,10 @@ public class RBFConfigKeys extends CommonConfigurationKeysPublic {
       FEDERATION_ROUTER_PREFIX + "heartbeat.interval";
   public static final long DFS_ROUTER_HEARTBEAT_INTERVAL_MS_DEFAULT =
       TimeUnit.SECONDS.toMillis(5);
+  public static final String DFS_ROUTER_HEALTH_MONITOR_TIMEOUT_MS =
```

Comment:
Let's use the getTimeDuration style.

Reply:
@goiri Thanks for your review. `getTimeDuration` will return a long value, but proxy `timeoutMs` need an Integer value. If we use `getTimeDuration`, we need to cast long to int. If you think it's ok, I will modify it.

Reply:
Let's do the getTimeDuration and cast it at the end.

Reply:
Copy, sir, I will fix it later.